### PR TITLE
docs: Adding clarity to SPI behaviour

### PR DESCRIPTION
**Pull request by garethtdavies**
In response to https://jel.ly.fish/437ac825-1c54-4ce8-8009-d6c6d91f994d. Also refreshed the configuration image.

Change-type: patch
Signed-off-by: Gareth Davies gareth@balena.io

### CODE REVIEW

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ To get you started, here is an [example][i2c-example] that uses i2c to communica
 
 ### SPI
 
-SPI is enabled by default and should work out of the box with the [spi node module][spi-npm].
+SPI is enabled by default on {{ $names.os.lower }}.
 
-For an example of this, check our this project: [digitiser][digitiser-link].
+For Node.js users it should work out of the box with the [spi node module][spi-npm]. For an example of this, check out this project: [digitiser][digitiser-link].
```
Comment (xginn8) [inline pages/learn/develop/hardware/i2c-and-spi.md]:
```suggestion
For Node.js applications it should work out of the box with the [spi node module][spi-npm]. For an example of this, check out this project: [digitiser][digitiser-link].
```

File: pages/learn/develop/hardware/i2c-and-spi.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ To get you started, here is an [example][i2c-example] that uses i2c to communica
 
 ### SPI
 
-SPI is enabled by default and should work out of the box with the [spi node module][spi-npm].
+SPI is enabled by default on {{ $names.os.lower }}.
```
Comment (rahul-thakoor) [inline pages/learn/develop/hardware/i2c-and-spi.md]: Maybe we can provide a bit context what it means enabled by default? some users might want to disable also? `dtparam=spi=on` in `config.txt`. Perhaps link to https://www.balena.io/docs/reference/OS/advanced/#setting-device-tree-overlays-dtoverlay-and-parameters-dtparam?

**Comment by garethtdavies** [comment]
@rahul-thakoor can you take another look at this when you get a chance?

**Comment by garethtdavies** [comment]
@balena-ci retest

**Pull request merged**